src/overlays/actors/ovl_Obj_Mure/z_obj_mure.c

```diff
@@ -14,7 +14,20 @@ void ObjMure_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjMure_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjMure_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+s32 ObjMure_GetMaxChildSpawns(ObjMure* this);
+void ObjMure_GetSpawnPos(Vec3f* outPos, Vec3f* inPos, s32 ptn, s32 idx);
+void ObjMure_SpawnActors0(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_SpawnActors1(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_SpawnActors(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_KillActorsImpl(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_KillActors(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_CheckChildren(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_InitialAction(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_CulledState(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_GroupBehavior0(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_GroupBehavior1(ObjMure* this, GlobalContext* globalCtx);
+void ObjMure_ActiveState(ObjMure* this, GlobalContext* globalCtx);
```

Comment:
Looks like many of the declarations aren't necessary so we would normally omit them. I think we could even avoid forward declaring the behavior functions if the function array was moved towards to the bottom of the file.